pkg/api/graph/graph.go

```diff
@@ -505,10 +506,10 @@ func ReverseGraphEdge(g Interface, head, tail graph.Node, edgeKinds sets.String)
 	return false
 }
 
-// AddReversedEdge adds a reversed edge for every passed edge and preserves the existing
-// edge. Used to convert a one directional edge into a bidirectional edge, but will
-// create duplicate edges if a bidirectional edge between two nodes already exists.
-func AddReversedEdge(g Interface, head, tail graph.Node, edgeKinds sets.String) bool {
+// AddReversedEdge adds a reference edge from head to tail without changing exiting edges.
```

Comment:
Can I petition for us to use unambiguous language like `from` and `to` or even  `u` and `v` which are understood ubiquitously across academia for the latter and in normal speech for the former? This is an unnecessary level of confusion here.